examples/rag/README.md

```diff
@@ -130,3 +130,29 @@ python examples/rag/eval_rag.py \
     --print_predictions \
     --recalculate \ # adding this parameter will force recalculating predictions even if predictions_path already exists
 ```
+
+# Use your own knowledge source
+
+RAG uses by default the enghlish wikipedia as a knowledge source, known as the 'wiki_dpr' dataset.
+You can build your own knowledge source using the `use_custom_knowledge_dataset.py` and use it for RAG.
+
+Let's say you have documents in tab-separated csv files with columns "title" and "text", then you can run for example:
```

Comment:
```suggestion
For instance, if documents are serialized as tab-separated csv files with the columns "title" and "text", one can use `use_own_knowledge_dataset.py` as follows:
```
Let's say you have documents in tab-separated csv files with columns "title" and "text", then you can run for example:
```